.github/workflows/pipeline-gha.yaml

```
@@ -113,7 +121,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ env.DOCKERHUB_ORG_NAME }}/${{ env.IMAGE_NAME }}
+          images: ${{ env.DOCKERHUB_ORG_NAME }}/${{ env.REPOSITORY_OWNER }}-${{ env.IMAGE_NAME_SUFFIX }}
```

```suggestion
          images: ${{ env.DOCKERHUB_ORG_NAME }}/${{ env.REPOSITORY_OWNER }}-${{ env.IMAGE_NAME_SUFFIX }}-gha
```

Adding the `gha` prefix here.